dashboard/app/models/school_stats_by_year.rb

```diff
@@ -58,7 +58,7 @@ def self.merge_from_csv(filename, options = {col_sep: "\t", headers: true, quote
     errored_schools = []
 
     ActiveRecord::Base.transaction do
-      CSV.read(filename, options).each do |row|
+      CSV.read(filename, **options).each do |row|
```

Comment:
I wasn't able to get any of the oneoff school_stats_by_year scripts to run without making this update (it was complaining about having 2 parameters when it only expected 1). I'm not quite sure why it [worked for Vijaya](https://github.com/code-dot-org/code-dot-org/pull/51491) without this change here but didn't work for me. Does this seem like a safe change?

Reply:
I think so. I _think_ this might be related: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/, which would make this a consequence of the Ruby upgrade

Reply:
Ahhh gotcha!